Core_Framework/Reporter/HtmlReport.cs

```diff
@@ -0,0 +1,179 @@
+﻿using AventStack.ExtentReports;
+using AventStack.ExtentReports.MarkupUtils;
+using AventStack.ExtentReports.Reporter;
+using Core_Framework.API_Core;
+using Core_Framework.Reporter.ExtentMarkup;
+using NUnit.Framework;
+
+namespace Core_Framework.Reporter;
+
+internal class HtmlReport
+{
+    private static ExtentReports? _report;
+    private static ExtentTest? _extentTestSuite;
+    private static ExtentTest? _extentTestCase;
+
+    // ------------------------------- CREATE EXTENTREPORT  -------------------------------
```

Comment:
If you want to separate the class part, please use the region instead

![image](https://user-images.githubusercontent.com/46469814/204457012-9dbc1b26-5758-4e2d-af9f-60e9ae779c6b.png)

Reply:
Done!